Created an Introduction page for the User Guide and the General Settings Section (#2024)
strapi/documentation: docusaurus/docs/user-docs/intro.md

Once you have logged in, you can access your Strapi dashboard:

<ThemedImage
alt="Homepage of the Admin Panel"
sources={{
    light: '/img/assets/getting-started/admin-panel-homepage.png',
    dark: '/img/assets/getting-started/admin-panel-homepage_DARK.png',
  }}
/>

If you are not sure where to begin on this User Guide, we suggest you to:

1. Go through the [User Guide Fundamentals](/user-docs/getting-started/user-guide-fundamentals).
2. Follow the [Setting up the Admin Panel](/user-docs/getting-started/setting-up-admin-panel) guide.
3. Continue your journey through the various User Guide sections. The ["What you will find here"](#what-you-will-find-here) section gives you an overview of the available topics.

## What you will find here

The table of content in this User Guide displays 7 main sections.

Clicking on any of the following cards will direct you to the introductory page for that section, with additional details and concepts:

<CustomDocCardsWrapper>
  <CustomDocCard emoji="📝" title="Content Manager" description="Manage and publish all content types created with the Content-type Builder." link="/user-docs/content-manager" />

>   <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, Manage and Publish Content Types including collection types." link="/user-docs/content-type-builder" />

Reviewer: There are 2 issues that you need to fix on this card's description:
- Style-related: capitalization and punctuation: it should be written 'Create, manage, and publish' (no uppercase on words inside a sentence except if they are Strapi features, plus usage of the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)
- Content-related: "Create, manage and publish content-types…" are you sure this is what the Content-Types Builder is about? 😄  I'm sure you can come up with a better definition of the CTB 🙂

>   <CustomDocCard emoji="🗃️" title="Media Library" description="Access all assets, upload assets, configure the view, and manage folders." link="/user-docs/media-library" />

Reviewer: I would suggest a shorter and slightly more accurate description. What do you think?
```suggestion
  <CustomDocCard emoji="🗃️" title="Media Library" description="Upload and manage all assets and organize them into folders." link="/user-docs/media-library" />
```

Author: Yeah this is better, thanks

  <CustomDocCard emoji="📅" title="Releases" description="Arrange entries into containers for publishing and unpublishing actions." />
  <CustomDocCard emoji="🔐" title="Users, Roles & Permission" description="Assign permission to roles, which are then associated with users." link="/user-docs/users-roles-permissions"/>
  <CustomDocCard emoji="🔗" title="Plugins" description="Find additional functionalities to extend and customize your applications." link="/user-docs/plugins/strapi-plugins" />

>   <CustomDocCard emoji="⚙️" title="General Settings" description="Access features needed set up your Strapi dashboard" link="/user-docs/settings/introduction" />

Reviewer: ```suggestion
  <CustomDocCard emoji="⚙️" title="General Settings" description="Access features needed set up your Strapi dashboard." link="/user-docs/settings/introduction" />
```
Missing period at the end of the sentence, for consistency with the other cards. 😄

Author: Thank you

The documentation for Strapi contains 3 main sections, accessible from the top navigation bar:

> - 🧑‍💻 The **[Developer Docs](/dev-docs/intro)** contain all the technical information related to the setup, advanced usage, customization, and update of your Strapi v5 application.

Reviewer: ```suggestion
- 🧑‍💻 The **[Developer Docs](/dev-docs/intro)** contain all the technical information related to the setup, advanced usage, customization, and update of your Strapi v4 application.
```

This was a typo I had in the Dev Docs too 🤦‍♂️ 😄 I've just [fixed it](https://github.com/strapi/documentation/commit/6caf6592824a50032f2e15a2f5a15657409c69bd) in the Dev Docs, could you please fix it here too?

Author: Okay, I've done that thank you.

1. Go through the [User Guide Fundamentals](/user-docs/getting-started/user-guide-fundamentals).
2. Follow the [Setting up the Admin Panel](/user-docs/getting-started/setting-up-admin-panel) guide.
3. Continue your journey through the various User Guide sections. The ["What you will find here"](#what-you-will-find-here) section gives you an overview of the available topics.

## What you will find here

The table of content in this User Guide displays 7 main sections.

Clicking on any of the following cards will direct you to the introductory page for that section, with additional details and concepts:

<CustomDocCardsWrapper>
  <CustomDocCard emoji="📝" title="Content Manager" description="Manage and publish all content types created with the Content-type Builder." link="/user-docs/content-manager" />
  <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, manage and publish more than one single entries." link="/user-docs/content-type-builder" />
  <CustomDocCard emoji="🗃️" title="Media Library" description="Upload and manage all assets and organize them into folders." link="/user-docs/media-library" />

>   <CustomDocCard emoji="📅" title="Releases" description="Arrange entries into containers for publishing and unpublishing actions." />

Reviewer 2: I believe there's an issue with the Releases card, the content is written in purple contrary to all other cards descriptions. Might be because the link is missing?

<img width="1097" alt="Screenshot 2024-02-28 at 09 58 55" src="https://github.com/strapi/documentation/assets/19183360/469e7a03-fa28-4b55-b79c-ec14e90011bd">

Author: Yes it was, I've included it now

  <CustomDocCard emoji="🔐" title="Users, Roles & Permission" description="Assign permission to roles, which are then associated with users." link="/user-docs/users-roles-permissions"/>

>   <CustomDocCard emoji="🔗" title="Plugins" description="Find additional functionalities to extend and customize your applications." link="/user-docs/plugins/strapi-plugins" />

Reviewer 2: Potentially just a question, not necessarily something to fix: why did you choose to link to the list of Strapi plugins and not to the introduction to plugins page, like you did for the other cards? 😊

<CustomDocCardsWrapper>
  <CustomDocCard emoji="📝" title="Content Manager" description="Manage and publish all content types created with the Content-type Builder." link="/user-docs/content-manager" />

>   <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, manage and publish more than one single entries." link="/user-docs/content-type-builder" />

Reviewer: I would suggest something like the following:

```suggestion
  <CustomDocCard emoji="📚" title="Content Type Builder" description="Build your data structure by creating and managing content-types" link="/user-docs/content-type-builder" />
```

Here's why I'm suggesting this:
- I feel like "publish" is partly wrong: though I can understand what you probably mean, "publishing" something is more related to the Content Manager than to the CTB. 
- I think it's important to keep "content-types" like in your original text, but also to quickly explain what it is.

What do you think? 🙂

Author: Hmm How about, "Create and manage content types for multiple entries to build your data structure"?

Reviewer: Sounds good too, except maybe for "multiple entries", what do you mean exactly with "multiple"? Because with the CTB you can create collection types (multiple entries) or single types (unique entry). 🤔
